src/fastcs/attributes.py

```diff
@@ -186,21 +189,26 @@ async def process(self, value: T) -> None:
         await self.update_display_without_process(value)
 
     async def process_without_display_update(self, value: T) -> None:
-        if self._process_callback is not None:
-            await self._process_callback(self._datatype.validate(value))
+        if self._process_callbacks:
+            await asyncio.gather(*[cb(value) for cb in self._process_callbacks])
 
     async def update_display_without_process(self, value: T) -> None:
-        if self._write_display_callback is not None:
-            await self._write_display_callback(self._datatype.validate(value))
+        self._datatype.validate(value)
```

Comment:
`validate` does type coercion of the value, so this still needs to be `value = ...validate(value)`